smart_pet_buddy/lib/homepage.dart

```diff
@@ -17,39 +19,101 @@ class HomePage extends StatefulWidget {
-        backgroundColor: Colors.green.shade400,
-        actions: <Widget>[
-          Builder(builder: (BuildContext context) {
-            return TextButton(
-              style: ButtonStyle(
-                  foregroundColor: MaterialStateProperty.all<Color>(
-                      Theme.of(context).buttonColor)),
-              onPressed: () async {
-                final User user = _auth.currentUser;
-                if (user == null) {
-                  ScaffoldMessenger.of(context).showSnackBar(const SnackBar(
-                    content: Text('No one has signed in.'),
+        appBar: AppBar(
+          title: Text('Home'),
+          backgroundColor: Colors.green.shade400,
+          actions: <Widget>[
+            Builder(builder: (BuildContext context) {
+              return TextButton(
+                style: ButtonStyle(
+                    foregroundColor: MaterialStateProperty.all<Color>(
+                        Theme.of(context).buttonColor)),
+                onPressed: () async {
+                  final User user = _auth.currentUser;
+                  if (user == null) {
+                    ScaffoldMessenger.of(context).showSnackBar(const SnackBar(
+                      content: Text('No one has signed in.'),
+                    ));
+                    return;
+                  }
+                  await _signOut();
+
+                  final String uid = user.uid;
+                  ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+                    content: Text('$uid has successfully signed out.'),
                   ));
-                  return;
-                }
-                await _signOut();
-
-                final String uid = user.uid;
-                ScaffoldMessenger.of(context).showSnackBar(SnackBar(
-                  content: Text('$uid has successfully signed out.'),
-                ));
-              },
-              child: const Text('Sign out'),
-            );
-          })
-        ],
-      ),
-      body: Controlpanel(),
+                },
+                child: const Text('Sign out'),
+              );
+            })
+          ],
+        ),
+        body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage('assets/images/homepage.jpg'),
+              fit: BoxFit.cover,
+            )
+          ),
+          child: Column(
+          children: [
+            Flexible(
+              flex: 1,
+              child: Center(
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    Text(
+                      'Welcome!',
+                      style: TextStyle(
+                        fontSize: 40,
+                      ),
+                    ),
+                    Text(
+                      'To start playing please connect to the car :)',
```

Comment:
Nice that you added this text! I think this is very helpful for the user and improves the user experience a lot.